src/gui/sdlmain.cpp

```diff
@@ -2352,19 +2336,26 @@ void GFX_EndUpdate(const uint16_t *changedLines)
 		// capture modes.
 		sdl.frame.present();
 	} else {
-		const auto frame_is_new = sdl.updating;
+		auto is_new_or_unskippable = []() {
+			static int16_t dupe_tally = 0;
+			if (sdl.updating || ++dupe_tally > sdl.frame.max_dupe_frames) {
+				dupe_tally = 0;
+				return true;
+			}
+			return false;
+		};
```

Comment:
Am I reading this correct, so say if we're in a 70 Hz DOS mode, max dupe frames will be 7.0. So if the program is displaying a still image, we're counting frames, then at the 8th frame we present the frame anyway? So we're effectively maintaining a steady 10 FPS even for still images, no matter what.

Also, I'm not sure what `unskippable` refers to here.